src/lib/api.js

```diff
@@ -1,6 +1,47 @@
 import axios from 'axios';
 
 const api = {
+    responseInterceptorId: null,
+    requestInterceptorId: null,
+    /**
+     * Adds a function to Axios that intercepts a response
```

Comment:
We should probably avoid surfacing the underlying XHR library in the docstrings. We could end up swapping it out for something else.